mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulHome.kt

```diff
@@ -35,6 +36,8 @@ private const val GameDefaultId = ""
 /** The route associated to new game button in play screen */
 private const val PrepareGameRoute = "prepare_game"
 
+private const val SpeechRecognitionRoute = "speech"
```

Comment:
Missing documentation, especially to indicate that this route is meant to be removed later.

Reply:
👍